src/Approximations/init_IntervalMatrices.jl

```diff
@@ -1,2 +1,3 @@
 eval(load_intervalmatrices_overapproximation())
 eval(load_intervalmatrices_overapproximation_expmap())
+eval(load_intervalmatrices_conversion())
```

Comment:
Really not important, but I found this confusing.
```suggestion
eval(load_intervalmatrices_overapproximation_matrixzonotope())
```